docs/content/latest/api/ysql/exprs/func_yb_hash_code.md

```diff
@@ -0,0 +1,195 @@
+---
+title: yb_hash_code() function [YSQL]
+headerTitle: yb_hash_code()
+linkTitle: yb_hash_code()
+description: Returns the partition hash code for a given set of expressions. 
+menu:
+  latest:
+    identifier: api-ysql-exprs-yb_hash_code
+    parent: api-ysql-exprs
+aliases:
+  - /latest/api/ysql/exprs/func_yb_hash_code
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+`yb_hash_code` is a function that returns the hash of a set of given input values using the hash function DocDB uses to shard its data. In effect, it provides direct access to the hash value of any given row of a YSQL table, allowing one to infer a row’s physical location. This enables an application to specify queries based on the physical location of a row or set of rows. A user’s newfound access to the physical location of a row should open up possibilities for more efficient queries.
+
+## Instructions
+`yb_hash_code` can be used anywhere a normal function can be used in a YSQL expression. There is no limitation to where it can be used. It takes in a variable number of parameters whose types must be allowed in YB primary keys. More formally,
+
+```
```

Comment:
Every code block needs a language. (`output` removes the copy button and doesn't source-highlight.)
````suggestion
```output
````